national/national_script.sh

```diff
@@ -126,6 +126,18 @@ done
 
 echo "DHIS2 core server is running."
 
+echo "Adding custom CSS..."
+
+curl -o ./scripts/style.css https://raw.githubusercontent.com/IntelliSOFT-Consulting/PSS-DHIS2-login/main/style.css
```

Comment:
Thanks, @nyatindopatrick. However, could you please update the seeding database instead of pushing this during deployment.